examples/tokens/index.ts

```diff
@@ -0,0 +1,66 @@
+import {Repo, getDefault, EcdsaKey, ChainTree, establishTokenTransaction, mintTokenTransaction,  sendTokenTransaction, receiveTokenTransactionFromPayload} from 'tupelo-wasm-sdk';
+const MemoryDatastore: any = require('interface-datastore').MemoryDatastore;
```

Comment:
Is there a reason to use `require` style import here instead of `import`?

Reply:
yeah because no types... I'll add a comment.

Reply:
@tobowers When starting to use interface-datastore myself in the other example, I realized why you're doing this :) You avoid needing type declarations, which was a pain to get right, so I switched to using `require` for it as well.